src/app/preparations/preparation/preparation.component.ts

```diff
@@ -77,7 +76,7 @@ export class PreparationComponent implements OnInit {
           .getPreparationById(this.id)
           .subscribe(preparation => this.preparation = preparation);
         this.heatmapService
-          .get(this.id, 100)
+          .get(this.id, 0)
```

Comment:
@gmrukwa Is there always heatmap with number 0?

Reply:
If not, we will have empty dataset. Much less likely than dataset with at least 101 heatmaps.